lib/decimal.ex

```diff
@@ -289,6 +299,10 @@ defmodule Decimal do
     context(%Decimal{sign: sign, coef: Kernel.abs(coef), exp: exp})
   end
 
+  def add(num1, num2) do
```

Comment:
@ericmj ok, let's extend this function to just accept integers. Do we want something like this:

```elixir
  def add(%Decimal{} = num1, num2) when is_integer(num2), do: add(num1, new(num2))

  def add(num1, %Decimal{} = num2) when is_integer(num1), do: add(new(num1), num2)

  def add(num1, num2) when is_integer(num1) and is_integer(num2), do: new(num1 + num2)
```

or there's a better way? The current implementation that just uses new/1 would also allow to pass floats and strings.